api/python/cell_census/src/cell_census/get_anndata.py

```diff
@@ -25,9 +25,7 @@ def get_anndata(
 ) -> anndata.AnnData:
     """
     Convience wrapper around soma.Experiment query, to build and execute a query,
-    and return it as an AnnData object.
-
-    [lifecycle: experimental]
```

Comment:
rm of lifecycle intentional?

Reply:
it was not removed.  Just merged with previous line.